src/ufs.rs

```diff
@@ -21,9 +25,21 @@ pub struct Ufs {
 
 impl Ufs {
 	pub fn open(path: &Path) -> Result<Self> {
-		let file = BlockReader::open(path)?;
+		let mut file = BlockReader::open(path)?;
+
+		let pos = SBLOCK_UFS2 as u64 + 1372;
```

Comment:
Where does the 1372 magic number come from?  It should be defined as a const.

Reply:
It comes from a test program like this:

``` c
#include <stdio.h>
#include "/usr/src/sys/ufs/ffs/fs.h"

int main (void)
{
    printf ("%zu\n", offsetof (struct fs, fs_magic));
    return 0;
}
```

But I agree, this should be a const

Reply:
Would it be easier to decode the entire Superblock?  That way you wouldn't have to calculate the exact offset of the magic field.

Reply:
Yes, but also no,
because but the whole point of this is to find out what endianness the FS is,
to then decode the superblock properly.

Reply:
Otherwise the superblock has to be decoded twice.